programmers/난이도별/level01.콜라츠_추측/Suhyeon.js

```diff
@@ -0,0 +1,24 @@
+function solution(num) {
+    var answer = 0;
+    
+    if(num==1){
+        return 0;
+    }
+    
+    do{
+        if(num%2==0)
+        {
+            num=num/2;
+            answer++;
+        }
+        else if(num%2!=0) 
+        {
+            num=num*3+1;
+            answer++;
+        }
```

Comment:
```suggestion
        num = num % 2 == 0? num / 2 : num * 3 + 1;
        answer++;
```
answer++; 은 if일때도, else일 때도 수행하기 때문에 if 문 밖에 넣어도 좋을 것 같아요!
그리고, 위에 쓴 if문을 삼항연산자로 표현하면 코드를 훨씬 간단하게 작성할 수 있어요.
참고해보시면 좋을 것 같아요~~😊